search/includes/classes/class-search.php

```diff
@@ -648,7 +648,14 @@ public function filter__ep_do_intercept_request( $request, $query, $args, $failu
 			$args['headers'] = [];
 		}
 
-		$args['headers'] = array_merge( $args['headers'], array( 'X-Client-Site-ID' => FILES_CLIENT_SITE_ID, 'X-Client-Env' => VIP_GO_ENV ) );
+		$args['headers'] = array_merge(
+			$args['headers'],
+			[
+				'X-Client-Site-ID' => FILES_CLIENT_SITE_ID,
+				'X-Client-Env' => VIP_GO_ENV,
+				'Accept-Encoding' => 'gzip, br',
```

Comment:
```suggestion
				'Accept-Encoding' => 'gzip, deflate',
```

Brotli is currently not supported by the libcurl version we use:

```
curl --compressed https://httpbin.org/brotli
curl: (61) Unrecognized content encoding type. libcurl understands deflate, gzip content encodings.
```